### PR TITLE
Cleanup Format-Hex

### PR DESCRIPTION
**Pull request by vexx32**
## PR Summary

Cleanup for code style prior to #8674

/cc @iSazonov ❤️ 

## PR Context  

Format-Hex can use a little love in the style department. Added some XML comments that were missing, tidied up existing ones, and made sure the code style overall is consistent with how we've been doing in recent PRs according to StyleCop and other linters.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [x] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs [hunk 1]
```diff
@@ -196,20 +197,20 @@ private void ProcessFileContent(string path)
...
 
-                        WriteHexidecimal(buffer.Slice(0, bytesRead), path, offset);
+                        WriteHexadecimal(buffer.Slice(0, bytesRead), path, offset);
 
                         offset += bytesRead;
                     }
                 }
             }
-            catch (IOException ioException)
+            catch (IOException fileException)
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs]: It is not always true :-) Please revert.

Comment (vexx32) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs]: CodeFactor complained that the `io` prefix looks too much like a Hungarian prefix, from memory. Is this simply not a concern?

Comment (iSazonov) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs]: We frequently use `catch (IOException e)`.
(I guess we will disable the check.)

**Review by iSazonov — APPROVED** [review]

**Review by daxian-dbw — APPROVED** [review]
LGTM. Thanks @vexx32 !

**Pull request merged**